### PR TITLE
Issue2431 disable send tip

### PR DESCRIPTION
**Pull request by jessopb**
Issue reported: Send button is not disabled on first view when amount is empty.
Now: Send button is disabled/grayed out while box is empty.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -185,7 +185,7 @@
     "yarn": "^1.3"
```
Comment (seanyesmunt) [inline package.json]: update this file to point to the new `lbry-redux` commit
`5fb6efae24205dfdfa133ca4f26bccce033570e1`

File: src/ui/component/dateTime/view.jsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // @flow
 import React from 'react';
 import moment from 'moment';
+import i18n from '../../i18n';
```
Comment (seanyesmunt) [inline src/ui/component/dateTime/view.jsx]: You should be able to just do `import i18n from 'i18n'`

File: package.json (hunk 1 above)
Comment (seanyesmunt) [reply package.json]: you can update `lbryinc` too since I just merged that PR.

`43d382d9b74d396a581a74d87e4c53105e04f845`

**Comment by jessopb** [comment]
Addressed in #2465 2465

**Pull request closed without merging**